extensions/amp-analytics/0.1/vendors.js

```diff
@@ -89,14 +89,23 @@ export const ANALYTICS_CONFIG = {
         'g3=${sponsorName}&' +
         'g4=${contentType}&' +
         'x=${scrollTop}&' +
+        'y=${scrollHeight}&' +
         'w=${screenHeight}&' +
```

Comment:
just curious, you are assigning screenHeight to w. Should it be h instead? also do you not care about the screen width?